.github/workflows/pages.yml

```diff
@@ -0,0 +1,49 @@
+# Simple workflow for deploying static content to GitHub Pages
+name: Deploy plugin docs to Pages
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
Remove once we're happy with the results at https://openrewrite.github.io/rewrite-maven-plugin
```suggestion
```